egs/ru_open_stt/asr1/conf/tuning/decode_pytorch_transformer_large.yaml

```diff
@@ -0,0 +1,6 @@
+beam-size: 2
```

Comment:
why did you use a very small beam?
Did you see some degradations or computational issues?

Reply:
Out of several tested beam sizes (20, 5, 2, 1) this one resulted the best WER, while others were slightly worse.